image-definitions/README.md

```diff
@@ -112,14 +115,27 @@ In many cases only single image should be published. In such cases perform publi
 
 ### Maintenance
 
-From time to time an update in upstream source, an addition of new container image, or a new vulnerability fix will require re-evaluation of image definitions. To perform difference analysis between upstream, set up a clean copy of upstream source in temporary directory and diff the contents with current image definitions.
+To create initial patch setup a clean copy of upstream source, make required adjustments that would resolve CVEs and produced the initial patch:
 
 For Kubeflow:
 
 ```
-mkdir ./update
-setup.sh ./update
-diff -r ./update/kubeflow kubeflow
+setup.sh .
+cd kubeflow
+git diff > ../kubeflow.patch
```

Comment:
I think a little more detail here would help.  Maybe something like the following?  

```suggestion
cd image-definitions
setup.sh .
cd kubeflow

# Manually make any modifications in the kubeflow directory, either by changing the files yourself
# or applying a patch

# Save a patch file describing your modified state
git diff > ../kubeflow.patch
```